explore-analyze/discover/discover-get-started.md

```diff
@@ -303,6 +305,35 @@ Note that in ES|QL mode, the **Documents** tab is named **Results**.
 
 Learn more about how to use ES|QL queries in [Using ES|QL](try-esql.md).
 
+### Run multiple explorations with tabs
+```{applies_to}
+stack: preview 9.2
+serverless: preview
+```
+**Discover** supports multiple tabs to help you explore different aspects of your data simultaneously. Each tab maintains its own independent state, including the query ({{esql}} or KQL), time range, filters, selected data source, columns and sort order you defined, and the active [context-aware experience](#context-aware-discover).
```

Comment:
```suggestion
**Discover** supports multiple tabs to help you explore different aspects of your data simultaneously. Each tab maintains its own independent state, including the query ({{esql}} or classic mode), time range, filters, selected data source, columns and sort order you defined, and the active [context-aware experience](#context-aware-discover).
```
Nit: probably fine as is since KQL is most common, but we also support Lucene queries in classic mode if we want to cover both here.